### DISCUSSION TITLE
Setting Netprofile function

### DISCUSSION THREAD

**Discussion by Kraego** [tbnobody/OpenDTU#899] [General]
Hello,

is it possible to **configure the netprofile** over opendtu, to AT (like described here: https://e4mobility.at/blog/hoymiles-netzprofil-richtig-einstellen)?

Thanks in advance.

**Comment by de-sascha** [comment]
Hi all I am currently also looking for this feature. Is this plan to build in future releases? 
Thanks a lot

I love this project.

**Comment by darkazar** [comment]
up
